chore: Add readme, license, and codeowners. (#1)
launchdarkly/open-sdk-specs: README.md

# LaunchDarkly SDK Specifications

> This repository contains open-sourced specifications for specific functionality of LaunchDarkly SDK implementations. These specifications aim to provide a deeper understanding of LaunchDarkly SDKs and assist in developing SDK extensions such as hooks.
> 
> These specifications focus on specific functionalities of the LaunchDarkly SDKs and are not a specification for an entire SDK.

Reviewer: ```suggestion
This repository contains open-sourced specifications for specific functionalities within LaunchDarkly SDK implementations. It does not contain a specification for an entire SDK.

These specifications provide a deeper understanding of LaunchDarkly SDKs and assist in developing SDK extensions such as hooks.
```
just a suggestion to remove the redundancy of the 1st and 3rd sentences
